modules/templates/util_render.go

```diff
@@ -118,10 +119,14 @@ func RenderIssueTitle(ctx context.Context, text string, metas map[string]string)
 }
 
 // RenderLabel renders a label
-func RenderLabel(ctx context.Context, label *issues_model.Label) template.HTML {
-	labelScope := label.ExclusiveScope()
+func RenderLabel(ctx context.Context, locale translation.Locale, label *issues_model.Label) template.HTML {
```

Comment:
Isn't the `locale` completely useless as  `ctx` already offers a `.Tr` method?
Or do we need a Gitea context for that?
And if yes, wouldn't it make more sense to refactor the existing parameter instead of adding a new one?

Reply:
I did that but I got a dependency cycle import ...

Reply:
Pity.
Sounds like we have a pending refactoring as it makes a lot more sense to have a base context in `modules` than the current workaround.

Reply:
```suggestion
// locale is needed due to an import cycle with our context providing the `Tr` function
func RenderLabel(ctx context.Context, locale translation.Locale, label *issues_model.Label) template.HTML {
```